ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -929,6 +948,22 @@ private void validateExplain(final PreparedStatement<Explain> statement) {
       explain(statement, scopedPropertyOverrides, ksqlConfig, executionSandbox);
     }
 
+    /**
+     * @deprecated `RUN SCRIPT` is deprecated since 5.2 and will be removed in the next major rel.
+     */
+    @SuppressWarnings({"MethodMayBeStatic", "DeprecatedIsStillUsed"})
+    private void validateRunScript(final PreparedStatement<RunScript> statement) {
```

Comment:
Perhaps log a message warning the user that RUN SCRIPT is deprecated? Ideally we'd include it in the response so they're more likely to see it, but that seems like it'd be a lot more work.

Reply:
Good idea - done.